reactbot.py

```diff
@@ -318,6 +323,106 @@ def react_announcement(self, message_json):
                 num = random.randrange(0, len(temp_list))
                 Bot.emoji_reaction(self, message_json["channel"], message_json["ts"], temp_list.pop(num))
 
+    def log_event(self, response):
```

Comment:
Can this not be abstracted to another file? I can see reasons to keep this out of Reactbot (as its a class function right now), and instead its own file full of functionality.